dohq_artifactory/admin.py

```diff
@@ -261,20 +262,104 @@ def _read_response(self, response):
         self._realm = response.get("realm")
 
     @property
-    def encryptedPassword(self):
-        if self.password is None:
+    def api_key(self):
+        """
+        Get an API key for the current user
+        :return: (str) API key
+        """
+        response = self._authenticated_user_request(
+            api_url="/api/security/apiKey", request_type=self._session.get
+        )
+        api_key = json.loads(response).get("apiKey", None)
+
+        return api_key
+
+    def api_key_create(self):
```

Comment:
Let's follow this name convention - `verb_object`, `create_api_key`
api_key_create looks uncommon. 

It wouldn't, if there was an ApiKeManager - `user.api_key.create`

Reply:
Idea here is to group all api_key methods together. First, dir() will be more readable
Second, some IDEs are not that smart, they will propose methods when you start typing

I would kindly ask to keep the names

Reply:
We shouldn't think about IDEs or other tools - think about how people will read code

- `user.api_key_create()`-  ok, we working with `user`, and their `api_key` and we'll `create`.... wait, what will we create? Now we need to apply backtracking to figure out what we'll create.

- `user.create_api_key()` - ok, we're working with a `user` and `create` an `api_key` - like a sentence! And it always should be this way

Reply:
Hello,
I liked your idea with class and introduced `_ApiKeyHandler`

you can see example in the doc